docs/config.md

```diff
@@ -131,3 +135,27 @@ The JSON file has the following fields:
   * `jira`: Optional Jira server and login information
   * `whitesource`: Optional whitesource server authentication information
   * `github_oauth`: Optional project specific GitHub OAuth token
+
+### create findings-[project].yaml files for each project
+
+The `findings-[project].yaml` file (where `[project]` is replaced with the project name used in the config file) contains information used in the generation of the HTML report.  The file contains a list of `findings` with the following fields:
+* `id` Id unique to each finding
+* `priority` Priority of the finding as displayed in the report
+* `matches-license` license name that this finding applies to as it appears in the fossology
+* `matches-path` optional file path that the finding applies to
+* `text` Text used for the finding in the HTML report
```

Comment:
one additional option -- `title` is an optional field that, if included, will set the title of the ticket in JIRA (where Jira integration is configured). See the Hyperledger findings file for an example.